dashboard/app/dsl/level_group_dsl.rb

```diff
@@ -27,6 +28,19 @@ def page
   end
 
   def level(name)
+    # Ensure level name hasn't already been used.
+    if @level_names.include? name
+      raise "Don't use the same level twice in a LevelGroup (#{name})."
+    end
+    @level_names << name
+
+    # Ensure level is appropriate type.
+    level = Level.find_by_name(name)
```

Comment:
This also ensures that levels exist, is this intentional? Probably a good idea... There's an argument for setting up the level group before the individual levels are done but they can leave them commented out or something if they want to do that.